geonetwork/gn-tool/gn-tool.sh

```diff
@@ -50,7 +51,13 @@ get_all_records() {
       gn_password_arg="--password $gn_password"
     fi
 
-    python ./get-uuids.py "$gn_addr/srv/eng/xml.search?fast=index&_isHarvested=n&from=1&to=2000" $gn_user_arg $gn_password_arg
+    if [[ $harvested_records != "n" ]]; then
```

Comment:
Not sure but won't this mean its now not possible to dump non-harvested records?

Reply:
Actually no - I can see you set it to 'n' by default

Reply:
This if statement is just confusing now and not required.

Reply:
Removed the if/else